### DISCUSSION TITLE
Option to apply local offset to all difficulties of a mapset

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#17458] [Ideas]
Having to manually go through every beatmap in a given mapset and apply the same offset to all of them was already a chore in stable, especially when you were still testing to see which offset was right. With lazer that issue seems all the more prevalent since, to my knowledge, there is no way to adjust the offset to exact values, with the slider  (and adjustment according to last play) seemingly being the only option to do so.

The only mention of this i could find was on an old stable forum post, but that post pitched the idea of having local offset apply to all difficulties in general. This is most likely not a good option, since different offset might be preferrable on different difficulties. I do however think that being able to, with on click, apply the offset of the current difficulty to all other difficulties would be a nice option to have.

**Comment by peppy** [comment]
Thinking about it, this should probably be the default behaviour...

**Reply by valflrt** [reply]
Maybe not the default behaviour because I noticed a small difference of offset between map (it is possible that depending on the map, the offset slightly changes).

**Reply by frenzibyte** [reply]
That indicates failure in the timing of said beatmap, a mistiming should always be the same across one track at the very least, unless the other diff uses a different track.
